.tekton/cli-build.yaml

```diff
@@ -351,24 +357,20 @@ spec:
           operator: in
           values:
             - "false"
-    - name: sast-shell-check
+    - name: clamav-scan
```

Comment:
The order of some tasks changed FYI. The GitHub diff viewer is not showing it well.